hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/devtools/sidecar/DtAnimatedWindow.kt

```diff
@@ -93,16 +74,14 @@ fun DtAnimatedWindow(
     // If you set `alwaysOnTop = true` for an invisible window in Linux, the window will lose its
     // ability to be always on top (even when it becomes visible again)
     val alwaysOnTopValue = alwaysOnTop && if (Os.currentOrNull() == Os.Linux) visible else true
+    val initialPosition = animateWindowPosition(windowState.position, size, true)
```

Comment:
I noticed being confused upon reading this line for the first time: 
Its called 'initialPosition' which lead me to think of it as 'not changing', 'just an initial value', however the RHS states that its literally anmiated?

Reply:
Yeah, it is indeed confusing. The RHS is not animated here, as we pass last argument `skipAnimation = true`. This is just an artifact of the past that I forgot to change, will update